README.md

```diff
@@ -1,2 +1,99 @@
 # open-sir
-Python SIR-x model implementation
+
+Open-SIR is an Open Source Python project for modelling pandemics and infectious diseases using Compartmental Models, such as the widely used [Susceptible-Infected-Removed (SIR) model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics). 
+
+The current stage of the software is *Alpha*.
+
+## Features
+- Model the dynamics of infectious diseases
+- Parameter fitting
+- Calculation of confidence intervals
+- CLI for interfacing with non Python environments (Bash, Node.JS, Matlab, etc).
```

Comment:
@jia200x how would this work? I know we have a CLI, but by this statement you mean something by the lines of:

```js
exec('opensir --help')
```

?

Reply:
Yes, exactly. Or someone could just generate CSV file to load them from Matlab